components/gc-dwnld/index.json-ld

```diff
@@ -0,0 +1,34 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Download link",
+		"fr": "Lien de téléchargement"
+	},
+	"description": {
+		"en": "Download link with custom styles.",
+		"fr": "Lien de téléchargement avec des styles personalisés."
+	},
+	"modified": "2023-04-20",
+	"componentName": "download-link",
```

Comment:
The component name must match the folder name or the other way around. Otherwise this will create a broken link in the main index page.
```suggestion
	"componentName": "gc-dwnld",
```